lib/fluent/plugin/in_kafka.rb

```diff
@@ -108,7 +108,13 @@ def configure(conf)
     @parser_proc = setup_parser
 
     if @use_record_time and @time_format
-      @time_parser = Fluent::TextParser::TimeParser.new(@time_format)
+      require 'fluent/version'
+      major, minor, patch = Fluent::VERSION.split('.').map(&:to_i)
+      if major > 0 || (major == 0 && minor >= 14 && patch >= 7)
```

Comment:
How about using `defined?` instead of each comparison?
Here is an example:

```
if defined?(Fluent::TimeParser)
```

Reply:
I get it. Fixed.